docs/examples/use_cases/pytorch/efficientnet/image_classification/dali.py

```diff
@@ -20,31 +20,34 @@
 from nvidia.dali.auto_aug import auto_augment, trivial_augment
 
 
-@pipeline_def(enable_conditionals=True)
-def training_pipe(data_dir, interpolation, image_size, output_layout, automatic_augmentation,
-                  dali_device="gpu", rank=0, world_size=1):
-    rng = fn.random.coin_flip(probability=0.5)
-
-    jpegs, labels = fn.readers.file(name="Reader", file_root=data_dir, shard_id=rank,
-                                    num_shards=world_size, random_shuffle=True, pad_last_batch=True)
-
-    if dali_device == "gpu":
-        decoder_device = "mixed"
-        resize_device = "gpu"
-    else:
-        decoder_device = "cpu"
-        resize_device = "cpu"
-
-    images = fn.decoders.image_random_crop(jpegs, device=decoder_device, output_type=types.RGB,
-                                           random_aspect_ratio=[0.75, 4.0 / 3.0],
-                                           random_area=[0.08, 1.0])
-
-    images = fn.resize(images, device=resize_device, size=[image_size, image_size],
-                       interp_type=interpolation, antialias=False)
+def processing_training_pipe(
```

Comment:
Nitpick: This name is quite ambiguous for me: are we processing the training pipe? Are we training to process? Are we processing and training?

Reply:
Maybe a docstring saying something like "This is the processing part of the training pipeline. Data loading is handled elsewhere" would help?